ui/orbital.c

```diff
@@ -55,8 +55,9 @@
 #define ORBITAL_HEIGHT 720
 
 typedef struct OrbitalUI {
+    /* window */
     bool active;
-    bool minimized;
+    enum WindowState {minimized, maximized} window_state;
```

Comment:
I actually have a question about this, should enum members also be in CamelCase?

Reply:
I wouldn't worry too much about style since we're going to apply clang-format sooner or later.

The thing that worries me is that a window can be in several states: minimized, maximized, windowed, fullscreen. When it comes to rendering, the last three are quite the same, so why even bothering with them? For all intents and purposes there's just minimized and non-minimized states, and at that point, wouldn't a `bool` just be better?

Reply:
That makes sense, yet at the same point making your code future proof is also a good idea, yeah i don't see a point for it myself at the moment either, also yeah, i didn't really think of the last two when making this even though they are quite similar, as i said below i thought of it as it would be one or the other, not both or none of them